extensions/resource-deployment/src/services/tools/dockerTool.ts

```diff
@@ -46,4 +46,12 @@ export class DockerTool extends ToolBase {
 	protected get versionCommand(): string {
 		return 'docker version --format "{{json .}}"';
 	}
+
+	get autoInstallSupported(): boolean {
+		return false;
+	}
+
+	public install(): Promise<void> {
+		return Promise.reject('Auto install is not supported for docker');
```

Comment:
why is this message different? is there another type of install that's supported?

Reply:
right, we don't need to support install of docker now as it requires more configuration and potentially restart.